### ISSUE TITLE
both building=yes and included building:part=yes are rendered on top of each other

### ISSUE THREAD

**Issue by joshinils** [OSMBuildings/OSMBuildings#228]
example:
https://www.openstreetmap.org/way/901822715
![image](https://user-images.githubusercontent.com/34302971/106374358-1343ad00-6383-11eb-8b79-ba853be043aa.png)

kinda the opposite of https://github.com/OSMBuildings/OSMBuildings/issues/209

I do not want to tag the outline building with `building:part=no` or `building:levels=0` since that makes no sense, and the wiki states that buildings containing parts should not be rendered.
> [When a building has any building:part=* areas, the building outline is not considered for 3D rendering.
](https://wiki.openstreetmap.org/wiki/Simple_3D_Buildings#Building_outlines)

I have added a relation to this https://www.openstreetmap.org/way/902165745 other house, but it hasn't updated yet so I don't yet know what that looks like.
Edit: no, the relation does not help :(

**Comment by Beakerboy** [comment]
I’ve been having the same issue. I was hoping adding wall=no would help, but it does not.

**Comment by natrius** [comment]
Its basically a duplicate of https://github.com/OSMBuildings/OSMBuildings/issues/163 but its true anyway, unfortunately. You can create a building-relation https://wiki.openstreetmap.org/wiki/Relation:building and it will get rendered, but in my opinion it would not be necessary if this problem with osmbuilding would get solved.

Btw. https://www.openstreetmap.org/way/902165745 seems to work properly with the relation? https://osmbuildings.org/?lat=52.63206&lon=13.29723&zoom=20.0&tilt=27&rotation=18 

If you look there https://osmbuildings.org/?lat=47.82930&lon=13.05306&zoom=18.4&tilt=28&rotation=-1 i used relation on one building in the east, the two other in the west don't - thats why they don't look good.

**Comment by joshinils** [comment]
no, https://www.openstreetmap.org/way/902165745 gets drawn in beige, not with a black roof, since all parts have a black roof i put the roof colour on the outline

**Comment by natrius** [comment]
Well... You tagged the building wrong. Remove everything from the outline `building=yes` and drop the tags onto the part `building:part=yes`. Then you should be alright.
